docs/tutorials/infrastructure/kne/srl-with-kne/srl-with-oc-services.md

```diff
@@ -0,0 +1,314 @@
+            "timestamp": 1665490620272174602,
+            "time": "2022-10-11T14:17:00.272174602+02:00",
+            "updates": [
+              {
+                "Path": "srl_nokia-system:system/srl_nokia-system-info:information/version",
+                "values": {
+                  "srl_nokia-system:system/srl_nokia-system-info:information/version": "v22.6.4-90-g4b19af2d95"
+                }
+              }
+            ]
+          }
+        ]
+        ```
+    === "Get using Openconfig YANG models"
+        When openconfig is not configured to be a default schema for gNMI server, users need to set the `origin` field of the path prefix to `openconfig` value:
+        ```bash
+        ❯ gnmic -a 172.18.0.50:9339 -u admin -p admin --skip-verify -e JSON_IETF \
+          get --prefix "openconfig:/" --path "/system/state/hostname"
+        [
+          {
+            "source": "172.18.0.50:9339",
+            "timestamp": 1665493415999114602,
+            "time": "2022-10-11T15:03:35.999114602+02:00",
+            "updates": [
+              {
+                "Path": "openconfig-system:system/state/hostname",
+                "values": {
+                  "openconfig-system:system/state/hostname": "srl1"
+                }
+              }
+            ]
+          }
+        ]
+        ```
+
+### gNOI
+
+<small>[:octicons-book-16: gNOI docs][gnmi-doc]</small>
+
+On SR Linux, gNOI service is enabled automatically once gNMI service is operational and share the same port `57400`. Although the same external post could have been used, to integrate with Ondatra test framework the separate service definition named `gnoi` with a separate `outside` port has been created.
+
+??? "Example"
+    gNOI service can be tested using [gnoic](https://gnoic.kmrd.dev) cli client.
+
+    ```bash
+    ❯ gnoic -a 172.18.0.50:9337 --skip-verify -u admin -p admin file stat --path /etc/os-release
+    +-------------------+-----------------+---------------------------+------------+------------+------+
+    |    Target Name    |      Path       |       LastModified        |    Perm    |   Umask    | Size |
+    +-------------------+-----------------+---------------------------+------------+------------+------+
+    | 172.18.0.50:57400 | /etc/os-release | 2021-09-14T06:32:07+02:00 | -rwxrwxrwx | -----w--w- | 21   |
+    +-------------------+-----------------+---------------------------+------------+------------+------+
+    ```
+
+### gRIBI
+
+<small>[:octicons-book-16: gRIBI docs][gribi-doc]</small>
+
+gRIBI server is enabled on a system level and in the `mgmt` network instance of SR Linux running on port `57401`. It is exposed to `9340` port for external connectivity as specified by the services configuration in the topology file.
+
+??? "Example"
+    gRIBI service can be tested using [gribic](https://gribic.kmrd.dev) cli client.
+
+    ```bash
+    ❯ gribic -a 172.20.20.2:9340 -u admin -p admin --skip-verify get --ns mgmt
```

Comment:
this address looks different from the others used in this guide

Reply:
fixed